addons/acr/units/CfgUnits.hpp

```diff
@@ -0,0 +1,207 @@
+//Units
+class GVAR(Rifleman): GVAR(Soldier_Base) {
+    scope = 2;
```

Comment:
Has it been checked these show in zeus? With base setting `scopeCurator = 0;` they might not show

Reply:
Does work in Zeus